src/main/java/org/openlmis/stockmanagement/controller/StockCardController.java

```diff
@@ -342,7 +346,26 @@ public ResponseEntity processStock(@PathVariable long facilityId,
         return OpenLmisResponse.success("Stock adjusted");
     }
 
-    private void filterEntries(StockCard stockCard, Integer entryCount) {
+
+
+    //Calls filterEntries() for each specified stockCard
+    private void filterEntries(List<StockCard> stockCards, Integer entryCount, boolean includeEmptyLots)
+    {
+        for (StockCard stockCard : stockCards) {
+            filterEntries(stockCard, entryCount, includeEmptyLots);
+        }
+    }
+
+    //Convenience method, calling TruncateStockCardEntries() and removeEmptyLotsFromStockCard()
+    private void filterEntries(StockCard stockCard, Integer entryCount, boolean includeEmptyLots)
+    {
+        TruncateStockCardEntries(stockCard, entryCount);
+        if(!includeEmptyLots)
+            removeEmptyLotsFromStockCard(stockCard);
+    }
+
+    //Filter stockCard.entries such that it contains only the first entryCount number of items
+    private void TruncateStockCardEntries(StockCard stockCard, Integer entryCount) {
```

Comment:
May want to lowercase first letter of method to match Java conventions and the other code.